src/SFA.DAS.EmployerFinance/Data/ITransactionRepository.cs

```diff
@@ -17,5 +17,6 @@ public interface ITransactionRepository
         Task<TransactionLine[]> GetAccountLevyTransactionsByDateRange(long accountId, DateTime fromDate, DateTime toDate);
         Task<string> GetProviderName(long ukprn, long accountId, string periodEnd);
         Task<TransactionDownloadLine[]> GetAllTransactionDetailsForAccountByDate(long accountId, DateTime fromDate, DateTime toDate);
+        Task<decimal> GetAllPaymentsForLastYear(long accountId);
```

Comment:
If we are returning just the number then wouldn't this be "GetTotalSpendForLastYear()"

Reply:
That's definitely a better name for it